iclr_wrap_up/utils.py

```diff
@@ -69,4 +69,33 @@ def is_dense_like(layer):
     Returns:
         True if layer has attribute 'kernel', False otherwise
     """
-    return hasattr(layer, 'kernel')
\ No newline at end of file
+    return hasattr(layer, 'kernel')
+
+
+def get_min_max(activations_summary, layer_number, neuron_number=None):
+    epochs_in_activation_summary = [int(epoch) for epoch in activations_summary]
+    epochs_in_activation_summary = np.asarray(sorted(epochs_in_activation_summary))
+
+    total_max = 0
+    total_min = 0
+    for epochs in epochs_in_activation_summary:
+        activations = activations_summary[f'{epochs}/activations']
+
+        if neuron_number is not None:
+            layer_activations = np.asarray(activations[str(layer_number)])
```

Comment:
in the if-else block, rather call a function with a descriptive name  that return current_min and current_max